src/objects/Hierarchy_Settings__c.object

```diff
@@ -390,6 +390,16 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Unique_Program_Plan_Enforced__c</fullName>
+        <defaultValue>true</defaultValue>
```

Comment:
Do we need to make changes to the install script to ensure that current customers, on upgrade to 1.7x, have this set to false by default, but new customers have it set to true by default?